checkpoint_quantum_firewall/assets/logs/checkpoint-quantum-firewall.yaml

```
@@ -0,0 +1,705 @@
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Source
+    path: src
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Source Machine Name
+    path: src_machine_name
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Status
+    path: checkpoint_firewall_status
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Subject
+    path: checkpoint_firewall_subject
+    source: log
+    type: string
+  - facetType: range
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Total Bytes
+    path: checkpoint_firewall_bytes
+    source: log
+    type: integer
+    unit:
+      family: bytes
+      name: byte
+pipeline:
+  type: pipeline
+  name: Checkpoint Quantum Firewall
+  enabled: true
+  filter:
+    query: "source:checkpoint-quantum-firewall"
+  processors:
+    - type: service-remapper
+      name: Define `product` as the official service of the log
+      enabled: true
+      meta:
+        tags: []
+      sources:
+        - product
+    - type: date-remapper
+      name: Define `time` as the official date of the log
+      enabled: true
+      meta:
+        tags: []
+      sources:
+        - time
+    - name: Lookup for `severity` to `checkpoint_firewall_severity` field
+      enabled: true
+      meta:
+        tags: []
+      source: severity
+      target: checkpoint_firewall_severity
+      lookupTable: |-
+        0, Info
+        1, Low
+        2, Medium
+        3, High
+        4, Critical
+        Info, Info
+        Low, Low
+        Medium, Medium
+        High, High
+        Critical, Critical
+      type: lookup-processor
+    - type: attribute-remapper
+      name: Map `subject` to `checkpoint_firewall_subject`
+      enabled: true
+      meta:
+        tags: []
+      sources:
+        - subject
+      sourceType: attribute
+      target: checkpoint_firewall_subject
+      targetType: attribute
+      preserveSource: false
+      overrideOnConflict: false
+    - type: attribute-remapper
+      name: Map `client_name` to `checkpoint_firewall_client_name`
+      enabled: true
+      meta:
+        tags: []
+      sources:
+        - client_name
+      sourceType: attribute
+      target: checkpoint_firewall_client_name
+      targetType: attribute
+      preserveSource: false
+      overrideOnConflict: false
+    - type: attribute-remapper
+      name: Map `action` to `checkpoint_firewall_action`
+      enabled: true
+      meta:
+        tags: []
+      sources:
+        - action
+      sourceType: attribute
+      target: checkpoint_firewall_action
+      targetType: attribute
+      preserveSource: false
+      overrideOnConflict: false
+    - type: attribute-remapper
+      name: Map `protocol` to `checkpoint_firewall_protocol`
```

The preferred way to namespace attributes is to separate the names with dots. 

Also, is it OK to drop the Quantum Part of the path ? The attribute name could also be `checkpoint.quantum.firewall.protocol` if we wanted to keep coherence with the name of the  integration and the facet group.

```suggestion
      name: Map `protocol` to `checkpoint.quantum.firewall.protocol`
```

---

Done